### PR TITLE
Adjust snapshot formatting tests

### PR DESCRIPTION
**Pull request by max-sixty**
These are not actually needed for multiline strings...

### CODE REVIEW

**Comment by max-sixty** [comment]
I _think_ this is correct — we don't need `r#"..."#` just because `...` has multiple lines — `"..."` is fine.

But it would be bad if I were wrong! So I'll leave this for a bit.

File: insta/src/snapshot.rs [hunk 1]
```diff
@@ -754,32 +754,34 @@ a
 b"[1..];
     assert_eq!(
         SnapshotContents(t.to_string()).to_inline(0),
-        "r#\"
+        r##""
```
Comment (max-sixty) [inline insta/src/snapshot.rs:757]: Some of these changes are unrelated FYI; trying to make the tests clearer

**Comment by mitsuhiko** [comment]
I would prefer to always use the raw strings because then the diffs are less invasive if stuff changes.

**Comment by max-sixty** [comment]
Merging the test changes, have reverted the functional changes
